docs/playbooks/telegraf.yml

```diff
@@ -7,6 +7,8 @@
 
   roles:
     - role: debops.telegraf/env
+      when:
+        telegraf__influxdb_enable == True
```

Comment:
To jest potrzebne?

Reply:
tak bo nie wygenerują ci się zmienne w defaults/main.yml dotyczące Influxdb